elastic/datadog_checks/elastic/elastic.py

```diff
@@ -42,6 +43,7 @@ class NodeNotFound(Exception):
 class ESCheck(AgentCheck):
     SERVICE_CHECK_CONNECT_NAME = 'elasticsearch.can_connect'
     SERVICE_CHECK_CLUSTER_STATUS = 'elasticsearch.cluster_health'
+    SERVICE_CHECK_INDEX_STATUS = 'elasticsearch.index_health'
```

Comment:
Are you submitting this as a service check? It looks like you are sending it as a gauge instead. Can you either remove this or send it as a service check? If you send it as a service check, remember to update `service_checks.json`

Reply:
Good catch! I'm removing.